README.md

```diff
@@ -339,7 +342,7 @@ func InjectGraphQlService(
 
 ### Database Modeling
 
-![Entity Relation Diagram](doc/eng/db/er-v1.jpg)
+![Entity Relation Diagram](doc/db/er.jpg)
 
 ### Feature Toggle
 
```

Comment:
The code example here is outdated.

Reply:
I guess, the developers can go through the project to understand the nuances of this feature. We can just mention such a feature exists in README.

Reply:
https://github.com/short-d/short/issues/759